q2_feature_table/_subsample_ids.py

```diff
@@ -20,8 +22,14 @@ def subsample_ids(table: biom.Table, subsampling_depth: int,
                          'elements on the desired axis. The maximum depth '
                          'is: %d.' % len(table.ids()))
 
+    # Generate a random seed if seed is None
+    if seed == "random":
+        rng = np.random.default_rng()
+        seed = rng.integers(0, 2 ** 32 - 1)
+
```

Comment:
This should all be able to go away since this happens in `Table.subsample` - right? 

```suggestion
```